addons/default/visiosoft/advs-module/src/Http/Controller/advsController.php

```diff
@@ -32,6 +32,7 @@
 use Visiosoft\LocationModule\Village\VillageModel;
 use Visiosoft\PackagesModule\Package\PackageModel;
 use Visiosoft\ProfileModule\Adress\Contract\AdressRepositoryInterface;
+use Visiosoft\SeoModule\Legend\Command\AddMetaData;
```

Comment:
if seo is not installed? @diashalabi

Reply:
https://github.com/openclassify/openclassify/pull/740#discussion_r511009308

Reply:
I tested if the module is not installed no error no problem